packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1711,8 +1712,35 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 return { ...attemptData, error };
             }
 
+            const summary = summarizeResult.summary.tree;
+            const summaryTree = summary[channelsTreeName];
+            let dataStoreCount = 0;
+            let summarizedUnreferencedDataStoreCount = 0;
+            let handleCount = 0;
+            if (summaryTree.type === SummaryType.Tree) {
+                if (summaryTree.tree !== undefined) {
+                    dataStoreCount = Object.keys(summaryTree.tree).length;
+                    for(const key of Object.keys(summaryTree.tree)) {
+                        const dataStore = summaryTree.tree[key];
+                        if (dataStore.type === SummaryType.Tree && dataStore.unreferenced === true) {
+                            summarizedUnreferencedDataStoreCount ++;
```

Comment:
Nit: we do not use spaces before ++ operator - searching for "[a-zA-Z0-9] \+\+" in whole repo gives me two hits using prefix notation only.